ProjectLighthouse/Serialization/LbpOutputFormatter.cs

```diff
@@ -26,12 +26,18 @@ protected override bool CanWriteType(Type type)
         if (isSerializable) return base.CanWriteType(type);
         Logger.Warn($"Unable to serialize type '{type?.Name}' because it doesn't extend ISerializable: (fullType={type?.FullName}", LogArea.Serialization);
         return false;
-    } 
+    }
 
-    public override async Task WriteResponseBodyAsync(OutputFormatterWriteContext context, Encoding selectedEncoding)
+    public async override Task WriteResponseBodyAsync(OutputFormatterWriteContext context, Encoding selectedEncoding)
```

Comment:
Not sure if this is some personal override that you have but official C# guidelines suggest that async should be the last modifier.
```suggestion
    public override async Task WriteResponseBodyAsync(OutputFormatterWriteContext context, Encoding selectedEncoding)
```